packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -77,23 +79,23 @@ export class BarGauge extends PureComponent<Props> {
   };
 
   render() {
-    const { onClick, className } = this.props;
+    const { onClick, className, theme } = this.props;
     const { title } = this.props.value;
     const styles = getTitleStyles(this.props);
 
     if (!title) {
       return (
-        <div style={styles.wrapper} onClick={onClick} className={className}>
+        <button style={styles.wrapper} onClick={onClick} className={cx(clearButtonStyles(theme), className)}>
```

Comment:
```suggestion
        <button type='button' style={styles.wrapper} onClick={onClick} className={cx(clearButtonStyles(theme), className)}>
```
This needs to have `button` type to prevent the default submit behavior in forms, quite unlikely for this specific component, but better be safe.